monitoring/pra/dashboard.py

```diff
@@ -0,0 +1,312 @@
+            "mongodb_oplog_stats_end",
+            "mongodb_ss_repl_lastWrite_lastWriteDate",
+        )
+    ]
+
+
+up = [
+    Stat(
+        title=title,
+        dataSource="${DS_PROMETHEUS}",
+        reduceCalc="last",
+        minValue="0",
+        maxValue=replicas,
+        noValue="0",
+        targets=[
+            Target(
+                expr='sum(' + metric + ')',
+            ),
+        ],
+        thresholdType="percentage",
+        thresholds=[
+            Threshold("red", 0, 0.0),
+            Threshold("yellow", 1, 50.0),
+            Threshold("green", 2, 100.0),
+        ],
+    )
+    for title, metric, replicas in [
+        ["KafkaConnect Source", Metrics.UP(Metrics.DR_SOURCE, job="${kafka_connect_src_job}"), 1],
+        ["KafkaConnect Sink", Metrics.UP(Metrics.DR_SINK, job="${kafka_connect_sink_job}"), 1],
+        ["Kafka", Metrics.UP('job="${kafka_instance}"'), 1],
+    ]
+]
+
+lastLifecycle = Stat(
+    title="Last lifecycle op",
+    description="Time since the last eligible op on the primary site.",
+    dataSource="${DS_PROMETHEUS}",
+    format=UNITS.CLOCK_MSEC,
+    noValue="-",
+    reduceCalc="last",
+    targets=[
+        Target(
+            expr="time() - " + Metrics.LIFECYCLE_LAST_TIMESTAMP(),
+        )
+    ],
+    thresholds=[
+        Threshold("#808080", 0, 0.0),
+        Threshold("green", 1, 0.0),
+        Threshold("super-light-yellow", 2, 1800.0),
+        Threshold("orange", 3, 3600.0),
+        Threshold("red", 4, 3700.0),
+    ],
+)
+
+lastMongodbWrite = Stat(
+    title="Last DR mongo write",
+    description="Time since the last mongo DB write on DR site.",
+    dataSource="${DS_PROMETHEUS}",
+    format=UNITS.CLOCK_MSEC,
+    noValue="-",
+    reduceCalc="last",
+    targets=[
+        Target(
+            expr="time() - " + Metrics.MONGODB_LAST_WRITE_TS(Metrics.DR_SINK),
+        )
+    ],
+    thresholds=[
+        Threshold("#808080", 0, 0.0),
+        Threshold("green", 1, 0.0),
+        Threshold("super-light-yellow", 2, 1800.0),
+        Threshold("orange", 3, 3600.0),
+        Threshold("red", 4, 3700.0),
+    ],
+)
+
+kafka_lag = TimeSeries(
+    title="Kafka Lag",
+    dataSource="${DS_PROMETHEUS}",
+    fillOpacity=30,
+    legendDisplayMode="list",
+    legendCalcs=["max", "mean"],
+    unit=UNITS.SECONDS,
+    targets=[Target(
+        expr="max(" + Metrics.KAFKA_CONSUMERGROUP_LAG() + ")",
+        legendFormat=" ",
+    )],
+)
+
+
+kafka_connect_lag_source, kafka_connect_lag_sink = [
+    TimeSeries(
+        title="Kafka Connect " + name + " Lag",
+        dataSource="${DS_PROMETHEUS}",
+        fillOpacity=30,
+        legendDisplayMode="list",
+        legendCalcs=["max", "mean"],
+        unit=UNITS.SECONDS,
+        targets=[Target(
+            expr="sum(" + metric + ")",
+            legendFormat=" ",
+        )],
+    )
+    for name, metric in [
+        ["Source", Metrics.KAFKA_CONNECT_SOURCE_TASK_RECORDS_LAG(Metrics.DR_SOURCE, job="${kafka_connect_src_job}")],
```

Comment:
Does this metric really make sense for source ? as it doesn't consume from Kafka.
A better metric for lag on source might be `kafka_connect_mongodb_source_task_metrics_latest_mongodb_time_difference_secs` ?